lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -84,7 +89,14 @@ click_secondary_menu:
   - selector:
       xpath: //a[@class='pf-c-nav__link' and text()='<secondary_menu>']
     op: click
-
+click_item_in_action_list:
```

Comment:
Are you looking for a similar action `click_one_dropdown_action`?

Reply:
Yes, updated.